src/sentry/integrations/slack/message_builder/notifications.py

```diff
@@ -1,109 +1,62 @@
-from typing import Any, Dict, List, Mapping, Union
+from __future__ import annotations
+
+from typing import Any, Mapping
 
 from sentry.integrations.slack.message_builder import SlackBody
 from sentry.integrations.slack.message_builder.base.base import SlackMessageBuilder
-from sentry.integrations.slack.message_builder.issues import (
-    SlackIssuesMessageBuilder,
-    build_attachment_title,
-    get_title_link,
-)
+from sentry.integrations.slack.message_builder.issues import SlackIssuesMessageBuilder
 from sentry.models import Team, User
-from sentry.notifications.notifications.activity.new_processing_issues import (
-    NewProcessingIssuesActivityNotification,
-)
-from sentry.notifications.notifications.activity.release import ReleaseActivityNotification
 from sentry.notifications.notifications.base import BaseNotification, ProjectNotification
-from sentry.notifications.utils import get_release
-from sentry.utils.http import absolute_uri
-
-from ..utils import build_notification_footer
 
 
-def build_deploy_buttons(notification: ReleaseActivityNotification) -> List[Dict[str, str]]:
-    buttons = []
-    if notification.release:
-        release = get_release(notification.activity, notification.project.organization)
-        if release:
-            for project in notification.release.projects.all():
-                project_url = absolute_uri(
-                    f"/organizations/{project.organization.slug}/releases/{release.version}/?project={project.id}&unselectedSeries=Healthy/"
-                )
-                buttons.append(
-                    {
-                        "text": project.slug,
-                        "name": project.slug,
-                        "type": "button",
-                        "url": project_url,
-                    }
-                )
-    return buttons
+def get_message_builder(klass: str) -> type[SlackNotificationsMessageBuilder]:
```

Comment:
What is `klass`? I don't understand the variable name.

Reply:
Interesting, I knew `class` was reserved but hadn't seen `klass`.

Reply:
I think the convention in python is usually `cls` actually, `klass` is a ruby-ism

See https://www.python.org/dev/peps/pep-0008/#function-and-method-arguments